### PR TITLE
more player stats lua bindings

### PR DESCRIPTION
**Pull request by USBMenus**
adds 5 more lua bindings in network
get_player_money
get_player_wallet
get_player_bank
get_player_language_id
get_player_language_name

### CODE REVIEW

File: src/lua/bindings/network.cpp [hunk 1]
```diff
@@ -234,6 +236,81 @@ namespace lua::network
...
+		}
+		return -1;
+	}
+
+	// Lua API: Function
+	// Table: network
+	// Name: get_player_language_name
+	// Param: pid: integer: Index of the player.
+	// Call get_player_language_name(playerID)
+	static std::string get_player_language_name(int pid)
```
Comment (maybegreat48) [inline src/lua/bindings/network.cpp:304]: Can't you do this in Lua?

Comment (gir489returns) [reply src/lua/bindings/network.cpp:304]: I made the same argument for literally any of these in his last PR.

Comment (USBMenus) [reply src/lua/bindings/network.cpp:304]: > Can't you do this in Lua?

yes, but that would require the globals to be updated every time there's an update, this way it's easier for scripts to stay up to date

Comment (maybegreat48) [reply src/lua/bindings/network.cpp:304]: >  this way it's easier for scripts to stay up to date

I'm talking about the language id to string conversion

Comment (USBMenus) [reply src/lua/bindings/network.cpp:304]: > I'm talking about the language id to string conversion

oh mb, yea you could create a table and use the id for the index, i can remove it if you want

Comment (gir489returns) [reply src/lua/bindings/network.cpp:304]: Same argument here applies as the globals, you could do it in Lua, but we should be consistent. If we're exposing crap internally to the Lua layer they could be doing as an assistant, then we should expose as much as possible.

**Review by gir489returns — APPROVED** [review]

**Pull request merged**